eos/b-decays/b-to-psd-nu-nu.cc

```diff
@@ -189,10 +200,14 @@ namespace eos
                 * sqrt_lambda / (power_of<3>(4.0 * M_PI * m_B));
 
             // first term in square brackets in [FLS:2021A], eq. (8)
-            const double contr_vector = lambda / 24.0 * f_p * f_p * std::norm(wc.cL() +  wc.cR());
+            const double contr_vector = lambda / 24.0 * f_p * f_p * std::norm(wc.cVL() +  wc.cVR());
+            // second line in [FLS:2021A], eq. (8) (ignoring the {bs} Wilson coefficients)
+            const double contr_scalar = q2 * power_of<2>((m_B2 - m_P2)/(m_b - m_s)) / 8 * f_0 * f_0 * std::norm(wc.cSL() +  wc.cSR());
```

Comment:
Coding style: please leave a single white space to the left and right of every binary operator, i.e.:
``(...) / (m_b - m_s)`` rather than ``(...)/(m_b - m_s)``.